client/src/Component/CommonComponent/Button/index.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import "./style.css";
+
+export const button = ({ className, onClick, name }) => {
```

Comment:
Good Job @Dalmano. The buttons look nice! 
I have a suggestion, though. Since you're exporting one thing, why not use `export default` instead?
```suggestion
export default ({ className, onClick, name }) => {
```